packages/client/src/components/header/header.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { FC, MouseEventHandler, ReactElement } from 'react';
 import { Link, NavLink } from 'react-router-dom';
-import { Button } from '@ui';
+import Button from '@/components/ui/button/button';
```

Comment:
Можно в директории ui/button добавить index.ts и экспортировать кнопку оттуда чтобы импорты былы красивее:
 ```suggestion
import Button from '@/components/ui/button';
```